src/renderer/entities/wallet/ui/WalletIcon/WalletIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { WalletType } from '@renderer/shared/core';
+import { Icon } from '@renderer/shared/ui';
+import { WalletIconBg, WalletIconNames } from '@renderer/entities/wallet/model/constansts';
```

Comment:
Better use relative path inside segments
```suggestion
import { WalletIconBg, WalletIconNames } from '../..';
```